aspnetcore/security/authorization/policies.md

```diff
@@ -278,13 +278,13 @@ The sample code on [AspNetCore.Docs.Samples](https://github.com/dotnet/AspNetCor
 
 :::code language="csharp" source="~/../AspNetCore.Docs.Samples/samples/aspnetcore-authz-with-ext-authz-service/Contoso.Security.API/appsettings.json":::
 
-4. Import the [ContosoAPI.postman_collection.json](https://github.com/dotnet/AspNetCore.Docs.Samples/blob/main/samples/aspnetcore-authz-with-ext-authz-service/ContosoAPI.postman_collection.json) file into Postman and configure an environment with the following:
+4. Import the [ContosoAPI.collection.json](https://github.com/dotnet/AspNetCore.Docs.Samples/blob/main/samples/aspnetcore-authz-with-ext-authz-service/ContosoAPI.collection.json) file and configure an environment with the following:
```

Comment:
Something is missing now -- import the file into what or where?

Reply:
Updated. See https://github.com/dotnet/AspNetCore.Docs/issues/33095